spec/01-unit/04-utils_spec.lua

```diff
@@ -8,10 +8,14 @@ describe("Utils", function()
 
   describe("is_valid_uuid()", function()
     it("validates UUIDs from jit-uuid", function()
-      assert.True(utils.is_valid_uuid("cbb297c0-a956-486d-ad1d-f9b42df9465a"))
-      assert.False(utils.is_valid_uuid("cbb297c0-a956-486d-dd1d-f9b42df9465a")) -- invalid variant
+      assert.True (utils.is_valid_uuid("cbb297c0-a956-486d-ad1d-f9b42df9465a"))
       assert.False(utils.is_valid_uuid("cbb297c0-a956486d-ad1d-f9b42df9465a"))
     end)
+    pending("invalidates UUIDs with invalid variants", function()
+      -- this is disabled because existing uuids in the database fail the check upon migrations
+      -- see https://github.com/thibaultcha/lua-resty-jit-uuid/issues/8
+      assert.False(utils.is_valid_uuid("cbb297c0-a956-486d-dd1d-f9b42df9465a")) -- invalid variant
```

Comment:
Let's add a test with this UUID to prove it is working with invalid varients, while keeping this one pending.
```lua
it("validates UUIDs with invalid variants for backwards-compatibility reasons", function()
  assert.True(utils.is_valid_uuid("cbb297c0-a956-486d-dd1d-f9b42df9465a"))
end)
```